frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/MyGitBranchCheckoutOperation.kt

```diff
@@ -0,0 +1,16 @@
+@file:Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
+
+package com.virtuslab.gitmachete.frontend.actions.common
+
+import com.intellij.openapi.project.Project
+import com.virtuslab.qual.guieffect.UIThreadUnsafe
+import git4idea.branch.GitNewBranchOptions
+import git4idea.repo.GitRepository
+import git4idea.ui.branch.GitBranchCheckoutOperation
+import java.util.Collections
+
+@UIThreadUnsafe
+fun performGitBranchCheckoutOperation(project: Project, gitRepo: GitRepository, startPoint: String, options: GitNewBranchOptions) {
```

Comment:
My idea was to move the call to Kotlin, I was hoping that it would handle the new optional parameter properly. Sadly, it fails with some other compatibility errors on the plugin verifier. Maybe it can be fixed (?)

The other option is to ~copy-paste~ reimplement the logic of `GitBranchCheckoutOperation::perform`. My worries are that there might be some other internal classes that are used inside - to be verified.

Reply:
Ha! at first glance, it seems a similar problem to what's been worked around with https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1314